src/styles.css

```
@@ -2,6 +2,10 @@
 @import 'tw-animate-css';
 @import 'shadcn/tailwind.css';
 
+.font-roboto-sans {
+  font-family: 'Roboto', ui-sans-serif, system-ui, sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)